charts/osm-arc/values.yaml

```diff
@@ -17,11 +17,13 @@ osm:
     enablePermissiveTrafficPolicy: true
     image:
       registry: mcr.microsoft.com/oss/openservicemesh
+      tag: v0.11.1
     enforceSingleMesh: true
+    enableReconciler: true
```

Comment:
Can you explain in the PR description why we are setting enableReconciler to true?